application/org.openjdk.jmc.feature.flightrecorder/feature.xml

```diff
@@ -199,4 +199,10 @@
          version="0.0.0"
          unpack="false"/>
 
+   <plugin
+         id="org.openjdk.jmc.flightrecorder.hierarchical-edge"
```

Comment:
I'd prefer if we'd skip the dash.

Reply:
Renamed to `dependencyview` since now we have two types of diagrams: chord and edge bundling.